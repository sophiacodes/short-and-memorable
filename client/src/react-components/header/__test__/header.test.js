import React from 'react';
import Header from '../';
import renderer from 'react-test-renderer';

it('renders <Header /> correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});