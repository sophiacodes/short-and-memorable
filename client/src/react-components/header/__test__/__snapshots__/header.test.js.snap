// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Header /> correctly 1`] = `
<header
  className="header"
>
  <h1
    className="title logo"
  >
    <span
      className="logo-left"
    >
      Short&
    </span>
    Memorable.
  </h1>
  <h2
    className="subtitle"
  >
    Word.
  </h2>
</header>
`;
